### ISSUE TITLE
[ACT, Help] Update the help content for the WCAG20_Object_HasText rule

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1189]
### Project

a11y checker extension

### Description

this is to update the existing help content for the [WCAG20_Object_HasText](https://github.com/IBMa/equal-access/blob/dev-1143/accessibility-checker-engine/help-v4/en-US/WCAG20_Object_HasText.html) rule to add a "What to do" section. see more details in [#1143](https://github.com/IBMa/equal-access/issues/1143) 

Rule branch: [dev-1143](https://github.com/IBMa/equal-access/tree/dev-1143)

**Comment by philljenkins** [comment]
**Discussion:** 
The computed ”**_accessible name_**” is all the Checker should care about for this fail test case. It needs a simple fix to the existing rule to either ignore the `<img>` content in the `<object>` or correctly check only the `<object>` itself and not its descendants for the object's valid computed accessible name.

Help would need an additional **_What to do_** to cover the case where the _accessible name_ (alt text) of the object itself if it's an image.

**Comment by philljenkins** [comment]
### What to do

* If the `<object>` element is a Java applet, add an alternative text description
* **Or**, if the `<object>` element is a MPEG movie, add an image with alternative text
* **Or**, if the `<object>` element itself is an image, add alt text to ensure the  _accessible name_ of the object is calculated correctly

**Comment by philljenkins** [comment]
Tested and ready for QA
![image](https://user-images.githubusercontent.com/16193609/211940056-fae56239-7aa4-48be-9eca-154b68656b26.png)

**Comment by eirckr** [comment]
Validated
